python/lsst/ts/adamSensors/mockModel.py

```diff
@@ -0,0 +1,89 @@
+from pymodbus.client.sync import ModbusTcpClient as ModbusClient
+from pymodbus.exceptions import ConnectionException, ModbusIOException
+
+
+class AdamModel:
+    """
+    Class that reads sensor voltage(s) through an ADAM 6024 device
+
+        Parameters
+        ----------
+        ip : string
+            the IP address of the ADAM 6024 controller
+        port : int
+            the port number for the ADAM 6024 controller
+
+        Attributes
+        ----------
+        client : ModbusClient
+            the pymodbus object representing the ADAM 6024
+    """
+
+    def __init__(self, log, simulation_mode=False):
+        self.client = None
+        self.clientip = None
+        self.clientport = None
+        self.log = log
+
+        self.range_size = 20
+        self.range_start = -10  # zero point offset for the ADAM device
+        self.simulation_mode = simulation_mode
+
+    def connect(self, ip, port):
+        self.clientip = ip
+        self.clientport = port
+        if self.simulation_mode:
+            self.client = None
+        else:
+            self.client = ModbusClient(self.clientip, self.clientport)
+
+    def disconnect(self):
+        self.client.close()
+
+    def read_voltage(self):
+        """reads the voltage off of ADAM-6024's inputs for channels 0-5.
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        -------
+        volts : List of floats
+            the voltages on the ADAM's input channels
+        """
+        if self.simulation_mode:
+            return [0, 0, 0, 0, 0, 1.25]
+
+        else:
+            try:
+                readout = self.client.read_input_registers(0, 8, unit=1)
+                return [self.counts_to_volts(r) for r in readout.registers]
+            except AttributeError:
+                # read_input_registers() *returns* (not raises) a
```

Comment:
Consider creating a ticket to handle this in the future then, mention the ticket here. 

```
# TODO: (DM-?????)
```